bundler/helpers/v1/spec/functions/dependency_source_spec.rb

```diff
@@ -47,7 +47,7 @@
       ])
     end
 
-    context "specified as the default source" do
+    context "with specified as the default source" do
```

Comment:
@thavaahariharangit same thing on a lot of these as well, this should really read

`when specified as the default source`